kubernetes/track.yml

```diff
@@ -56,7 +56,8 @@ challenges:
   teaser: Use a NodePort to expose the NGINX webserver
   notes:
   - type: text
-    contents: You've just deployed NGINX. To view the deployment, create a NodePort service for it.
+    contents: You've just deployed NGINX. To view the deployment, create a NodePort
```

Comment:
do we want the newline?

Reply:
Probably not, but this was the remote state